webapp/src/pages/Settings.tsx

```diff
@@ -164,7 +164,7 @@ const Settings: React.FC = () => {
     />
   );
 
-  const displayArgumentsList = (
+  const displayKeyArgumentsList = (
```

Comment:
Since we are not using this function for lists anymore, we can remove `List` from its name.
Also, in Python, `kwargs` stands for "Keyword Arguments", not "Key Arguments", so I suggest being consistent here:
```suggestion
  const displayKeywordArguments = (
```